docs/en-US/component/autocomplete.md

```diff
@@ -110,3 +110,4 @@ autocomplete/custom-loading
 | loading          | remote search loading indicator             | ^[object]`Ref<boolean>`                   |
 | popperRef        | el-tooltip component instance               | ^[object]`Ref<ElTooltipInstance>`         |
 | suggestions      | fetch suggestions result                    | ^[object]`Ref<record<string, any>>`       |
+| getData          | run load data                               | ^[Function]`(queryString: string) => void`|
```

Comment:
It would be a good idea to mark in which version it will be available.
```suggestion
| getData ^(2.8.3) | run load data                               | ^[Function]`(queryString: string) => void`|
```

Reply:
Done